src/sections/stats/StatsPage.utils.ts

```diff
@@ -72,7 +72,7 @@ export const useOmnipoolAssetDetails = () => {
     ...positions,
     ...omnipoolAssetBalances,
   ]
-  const isInitialLoading = queries.some((q) => q.isInitialLoading)
```

Comment:
I'd leave the isInitialLoading, because there is a bug in the useQuery, when the query is disabled it the isLoading state is true

Reply:
I thought we want to display skeletons if dependencies are not ready (=query disabled). I wouldn't call it a bug, it's expected behaviour mentioned in documentation.

Reply:
https://github.com/galacticcouncil/HydraDX-ui/pull/518#discussion_r1275879055